Makefile

```diff
@@ -54,12 +54,10 @@ run:
 	docker-compose -p edgex -f docker-compose${NO_SECURITY}${UI}${ARM64}.yml up -d ${SERVICES}
 
 down:
-	docker-compose -p edgex -f docker-compose.yml -f docker-compose-no-secty-with-ui.yml down
+	docker-compose -p edgex -f docker-compose.yml -f docker-compose-no-secty-with-ui.yml down ${CLEAN}
```

Comment:
Anytime there is an optional make variable like this, I recommend defining it in the variables section at the top like so
```
CLEAN ?=
```
otherwise you will get a warning if you enable the warn on undefined variables flag.